src/forms/RadioButtonGroup.jsx

```diff
@@ -1,73 +1,68 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 
-import Flex from '../layout/Flex';
-import FlexItem from '../layout/FlexItem';
-
 /**
  * @class RadioButtonGroup
  */
 export default class RadioButtonGroup extends PureComponent {
 	constructor(props) {
 		super(props);
-		this.state = {
-			selectedValue: props.selectedValue,
+
+		const initialState = {
+			value: props.selectedValue,
```

Comment:
there's some problematic stuff in how this component is written, but _some_ of the is problems were there in the initial implementation so I'm not sure how much we can 'fix'.

In general, it's [not a good idea to copy props into state](https://overreacted.io/writing-resilient-components/#principle-1-dont-stop-the-data-flow) - in this case, every time the parent re-renders, the current `state.value` value will be overwritten with `props.selectedValue`, which would cause problems.

The updates to the code also overwrite the `this.state[value]` map on every render, which probably isn't what is intended.